### PR TITLE
fixes RHOARDOC-1216

### PR DESCRIPTION
**Pull request by inoxx03**
Reusability changes for the App from scratch content.
+ it makes the starter app development assembly shareable by Vert.x  and Spring Boot.
+ now with comments providing parameterization reference
+ still uses way too many parameters, but works :)

### CODE REVIEW

**Comment by inoxx03** [comment]
@rhoads-zach @tradej Please let me know what you think.

File: docs/vertx-runtime/master.adoc [hunk 1]
```diff
@@ -255,11 +255,21 @@ include::topics/secured-mission-resources.adoc[leveloffset=+3]
 //Minimum Viable App
 //Add-ons
 //Building your app
+// TODO: make content properly reusable
+//creating an app from scratch
```
Comment (Ladicek) [inline docs/vertx-runtime/master.adoc]: Does the TODO still make sense? :-)

Comment (inoxx03) [reply docs/vertx-runtime/master.adoc]: @Ladicek Oh, right. That one is no longer necessary. Guess I should remove that :)

File: docs/topics/resources/vertx-starter-example-pom.xml [hunk 2]
```diff
@@ -12,8 +12,8 @@
   <description>Example application using RHOAR Vert.x</description>
 
   <properties>
-    <vertx.version>3.4.2.redhat-009</vertx.version>
-    <vertx-maven-plugin.version>1.0.13</vertx-maven-plugin.version>
+    <vertx.version>{VertXVersion}</vertx.version>
+    <vertx-maven-plugin.version>{VertXMvnPluginVersion}</vertx-maven-plugin.version>
```
Comment (Ladicek) [inline docs/topics/resources/vertx-starter-example-pom.xml]: I see `VertXVersion` defined in `document-attributes.adoc`, but where is `VertXMvnPluginVersion` defined?

Comment (inoxx03) [reply docs/topics/resources/vertx-starter-example-pom.xml]: @Ladicek Thank you for calling this out. I've actually noticed that myself a while ago and added the `VertxMvnPluginVersion` variable to`document attributes`, which store the VMP version.

Comment (Ladicek) [reply docs/topics/resources/vertx-starter-example-pom.xml]: Cool, thanks!

**Pull request closed without merging**